Enabling caching in development, and discard on each request. (#372)
github/view_component: CHANGELOG.md

# master

> * Enable caching in development, but discard on each request. Prevents unnecessary recompiles during the same request.

Reviewer: ```suggestion
* Cache components per-request in development, preventing unnecessary recompilation during a single request.
```
